docs/output.md

```diff
@@ -329,14 +329,15 @@ Files in these two folders contain all contigs of an assembly.
 
 ### Bin sequencing depth
 
-For each genome bin the median sequencing depth is computed based on the corresponding contig depths given in `GenomeBinning/[assembler]-[sample/group]-depth.txt.gz`.
+For each genome bin the median sequencing depth is computed based on the corresponding contig depths.
 
 <details markdown="1">
 <summary>Output files</summary>
 
 - `GenomeBinning/`
   - `bin_depths_summary.tsv`: Summary of bin sequencing depths for all samples. Depths are available for samples mapped against the corresponding assembly, i.e. according to the mapping strategy specified with `--binning_map_mode`. Only for short reads.
   - `[assembler]-[sample/group]-binDepths.heatmap.png`: Clustered heatmap showing bin abundances of the assembly across samples. Bin depths are transformed to centered log-ratios and bins as well as samples are clustered by Euclidean distance. Again, sample depths are available according to the mapping strategy specified with `--binning_map_mode`.
+  - `[assembler]-[sample/group]-depth.txt.gz`: Per-contig fold-depth summary statistics for raw contig bins.
```

Comment:
```suggestion
```

This file is already described above as output in [Contig sequencing depth](https://github.com/nf-core/mag/blob/dev/docs/output.md#contig-sequencing-depth)